dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -50,7 +50,7 @@ def replace_hostname(url)
 
 When /^I close the React alert$/ do
   steps %q{
-    When I press the first ".react-alert button" element
+    When I click selector ".react-alert button span"
```

Comment:
wondering now if this would even work with press as long as I add span to the selector. giving that a try